p2panda-rs/src/schema/schema_id.rs

```diff
@@ -11,6 +11,12 @@ use crate::document::DocumentViewId;
 use crate::operation::OperationId;
 use crate::schema::error::SchemaIdError;
 
+/// Schema id for _schema definition_ schema
+pub(super) const SCHEMA_DEFINITION_ID_STR: &str = "schema_definition_v1";
+
+/// Schema id for _schema field definition_ schema
+pub(super) const SCHEMA_FIELD_DEFINITION_ID_STR: &str = "schema_field_definition_v1";
+
```

Comment:
I added these mostly because working on key groups I saw that we will quickly have a lot of hard coded schema id strings in this file at multiple places once we start adding more system schemas. So not solving a current problem, more about future proofing.